health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -861,7 +921,12 @@ export const UICustomizations = {
               }
             };
   
+            const handleToast = () => {
+              setShowToast(false);
+            };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)